SwiftI18n/Classes/Views/SwiftUI/LocText.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  SwiftUIView.swift
+//
+//
+//  Created by Antonijo Bezmalinovic on 14.04.2023..
+//
+
+import SwiftUI
+
+enum SwiftI18nKey: EnvironmentKey {
+    static var defaultValue: String = I18nManager.instance.language
+}
+
+public extension EnvironmentValues {
+    
+    var language: String {
+        get { self[SwiftI18nKey.self] }
+        set { self[SwiftI18nKey.self] = newValue }
+    }
+}
+
+public struct LanguageModifier: ViewModifier {
+    
+    @State private var language = I18nManager.instance.language
+    
+    public init(){}
+
+    public func body(content: Content) -> some View {
+        content
+            .onReceive(
+                NotificationCenter.default.publisher(for: .loc_LanguageDidChangeNotification).receive(on: DispatchQueue.main)
+            ) { _ in
+                language = I18nManager.instance.language
+            }
+            .environment(\.language, language)
+    }
+    
+}
+
+public struct LocText<S>: View where S: RawRepresentable, S.RawValue == String {
+    
+    public let key: S
```

Comment:
It could be easier and maybe a bit more correct to make key string, and then you can easily make make an extension, so it doesn't have to exclusively depend on polyglot.
```swift
public extension LocText {

init<S: RawRepresentable>(_ key: S) where S.RawValue == String {
self.key = key.rawValue
}

}
```